src/components/MonthlyDataDisplay/MonthlyDataDisplay.js

```diff
@@ -19,27 +19,30 @@ function MonthlyDataDisplay({
     
   const storeVariable = useStore((state) => state.setMonthlyIndicator);
   const variable = useStore((state) => state.monthlyIndicator);
+  const viewOutletIndicators = useStore((state) => state.viewOutletIndicators);
+
   
   function dontSelectVariable(event) {
     // no-op, as we are not using cascading selection
   }
 
   useEffect(() => {
     if (region && variable && rasterMetadata) {
+      const area = viewOutletIndicators ? JSON.parse(region.outlet) : region.boundary;
```

Comment:
The `area` computation is repeated 3 different places. Consider factoring it out to a function?

It's tempting to consider DRYing up the other redundancies between these data displays. That might more trouble than it's worth and then having to undo it when they diverge at some future point. But worth a moment's thought.